.github/workflows/test_pipeline.yml

```
@@ -26,23 +26,19 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python 3.7
+      - name: Set up Python3
         uses: actions/setup-python@v1
         with:
-          python-version: 3.7
+          python-version: '3.x'
 
-      - name: Display Python version
+      - name: Display Python version and path
         run: python -c "import sys; print(sys.version)"
       
       - name: Install dependencies
         run: |
+          sudo apt-get install libsnappy-dev
```

Should you pass the `-y` flag to prevent it from prompting?

```suggestion
          sudo apt-get install libsnappy-dev -y
```